run.sh

```diff
@@ -9,4 +9,4 @@ nohup java -jar selenium-server-standalone-2.53.0.jar &
 #################
 #Start the agent#
 #################
-PYTHONPATH=${PYTHONPATH}:. python agent/agentCore.py
+nohup PYTHONPATH=${PYTHONPATH}:. python agent/agentCore.py &
```

Comment:
it runs it in a unbounded state. so that if the terminal gets closed the process doesnt get closed